scripting/ttt_item_ump.sma

```diff
@@ -92,6 +93,20 @@ public cswa_damage(weapon_id, victim, attacker, Float:damage)
 	if(get_weapon_edict(weapon_id, REPL_CSWA_ITEMID) == g_iItemID)
 	{
 		new Float:angles[3];
+		new colors[3];
+		new VictimState = ttt_get_playerstate(victim);
+		if(VictimState == PC_DETECTIVE || VictimState == PC_TRAITOR || VictimState == PC_INNOCENT)
+			{
+				if(VictimState == PC_TRAITOR)
+				colors = {255,0,0};
+				if(VictimState == PC_DETECTIVE)
+				colors = {0,0,255};
+				if(VictimState == PC_INNOCENT)
+				colors = {0,255,0};
+				
+				set_user_rendering(victim,kRenderFxGlowShell,colors[0],colors[1],colors[2],kRenderNormal,25);
+				set_task( 2.5, "remove_glowump", victim );
+			}
```

Comment:
Many people wanted UMP to be highlighted so i did that and tested on my server and got a great response, everyone liked it :P